beekeeper-path-scheduler-apiary/src/test/java/com/expediagroup/beekeeper/scheduler/filter/WhitelistedListenerEventFilterTest.java

```diff
@@ -75,44 +55,43 @@ public void filterAlterTableEvent() {
                            "DROP_TABLE",
                            "Alter_Table , Create_Table, Drop_table" })
   public void filterWhitelistedEvent(String whitelist) {
-    when(dropTableEvent.getEventType()).thenReturn(EventType.DROP_TABLE);
-    when(dropTableEvent.getTableParameters())
+    when(listenerEvent.getEventType()).thenReturn(EventType.DROP_TABLE);
+    when(listenerEvent.getTableParameters())
       .thenReturn(Map.of(BEEKEEPER_HIVE_EVENT_WHITELIST, whitelist));
-    boolean filter = listenerEventFilter.filter(dropTableEvent);
+    boolean filter = listenerEventFilter.filter(listenerEvent);
     assertThat(filter).isFalse();
   }
 
   @ParameterizedTest
   @ValueSource(strings = { "",
                            "  ",
-                           "create_table,alter_table",
+                           "create_table,drop_table",
                            "drop_table alter_table",
-                           "drop table,create_table",
-                           "drop-table" })
+                           "alter table,create_table",
+                           "alter-table" })
   public void filterNotWhitelistedEvent(String whitelist) {
-    when(dropTableEvent.getEventType()).thenReturn(EventType.DROP_TABLE);
-    when(dropTableEvent.getTableParameters())
+    when(listenerEvent.getEventType()).thenReturn(EventType.ALTER_TABLE);
+    when(listenerEvent.getTableParameters())
       .thenReturn(Map.of(BEEKEEPER_HIVE_EVENT_WHITELIST, whitelist));
-    boolean filter = listenerEventFilter.filter(dropTableEvent);
+    boolean filter = listenerEventFilter.filter(listenerEvent);
     assertThat(filter).isTrue();
   }
 
   @Test
   public void filterNullTableParameters() {
-    when(dropTableEvent.getEventType()).thenReturn(EventType.DROP_TABLE);
-    when(dropTableEvent.getTableParameters())
+    when(listenerEvent.getTableParameters())
       .thenReturn(null);
-    boolean filter = listenerEventFilter.filter(dropTableEvent);
-    assertThat(filter).isTrue();
+    boolean filter = listenerEventFilter.filter(listenerEvent);
+    assertThat(filter).isFalse();
   }
 
   @Test
   public void filterEmptyTableParameters() {
-    when(dropTableEvent.getEventType()).thenReturn(EventType.DROP_TABLE);
-    when(dropTableEvent.getTableParameters())
+    when(listenerEvent.getEventType()).thenReturn(EventType.DROP_TABLE);
```

Comment:
Can you make these parametersized on the `EventType`.
So one test for accepting AlterTable and AlterPartition and one for not accepting all the other events when the default is set?